lib/graphql/middlewares/send-email.ts

```diff
@@ -86,29 +90,36 @@ export const onEventCreationComplete: MiddlewareFn<TContext> = async ({ args, co
         form_name: 'Event Check-in Generator',
         url: `https://portal.acmutd.co/checkin/${createdEvent.id}`,
       }),
-      await sendEventToDiscordGuildEvent(createdEvent)
+      await sendEventToDiscordGuildEvent(createdEvent),
+      await addEventToGCal(createdEvent),
```

Comment:
What do you think about wrapping these 2 function calls within `Promise.all`?

Reply:
They should be in the `Promise.all`

Reply:
Sorry, GitHub hid the unchanged part so I didn't realize that it's already there.